cmake/modules/SwiftComponents.cmake

```diff
@@ -122,3 +122,29 @@ function(swift_install_in_component component)
     install(${ARGN})
   endif()
 endfunction()
+
+function(swift_install_symlink_component component)
+  cmake_parse_arguments(
+      ARG # prefix
+      "" "LINK_NAME;TARGET;DESTINATION" "" ${ARGN})
```

Comment:
I'm happy to add the full set of comments in. Also, I realized why CMake doesn't support the behavior you suggested, and I think it would be very difficult for us to support it.

Take for example this situation:

swift_parse_args(PREFIX FOO MULTIVALUE_OPTIONS OPTIONS)

What is the expected behavior of that? It could be specifying a multi-value option named "OPTIONS", or defining MULTIVALUE_OPTIONS and OPTIONS as empty.